tests/testing/LocalBearTestHelperTest.py

```diff
@@ -126,6 +126,7 @@ def create_arguments(filename, file, config_file):
                 raise AssertionError
         except AssertionError as ex:
             self.assertIn('The program yielded the following output:', str(ex))
+            self.assertIn('Arguments:', str(ex))
```

Comment:
You could also assert for the arguments in this case ;)

Reply:
done